store/copr/coprocessor.go

```diff
@@ -1158,6 +1169,11 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 	if len(worker.req.MatchStoreLabels) > 0 {
 		ops = append(ops, tikv.WithMatchLabels(worker.req.MatchStoreLabels))
 	}
+	if task.redirect2Replica != nil {
+		req.ReplicaRead = true
```

Comment:
Should we move the setting `busyThreshold=0` here so none of the changes would be missing?